docs-devsite/vertexai-preview.schema.md

```diff
@@ -0,0 +1,264 @@
+Project: /docs/reference/js/_project.yaml
+Book: /docs/reference/_book.yaml
+page_type: reference
+
+{% comment %}
+DO NOT EDIT THIS FILE!
+This is generated by the JS SDK team, and any local changes will be
+overwritten. Changes should be made in the source code at
+https://github.com/firebase/firebase-js-sdk
+{% endcomment %}
+
+# Schema class
+Parent class encompassing all Schema types, with static methods that allow building specific Schema types. This class can be converted with JSON.stringify() into a JSON string accepted by Vertex REST endpoints. (This string conversion is automatically done when calling SDK methods.)
+
+<b>Signature:</b>
+
+```typescript
+export declare abstract class Schema implements SchemaInterface 
+```
+<b>Implements:</b> [SchemaInterface](./vertexai-preview.schemainterface.md#schemainterface_interface)
+
+## Constructors
+
+|  Constructor | Modifiers | Description |
+|  --- | --- | --- |
+|  [(constructor)(schemaParams)](./vertexai-preview.schema.md#schemaconstructor) |  | Constructs a new instance of the <code>Schema</code> class |
+
+## Properties
+
+|  Property | Modifiers | Type | Description |
+|  --- | --- | --- | --- |
+|  [description](./vertexai-preview.schema.md#schemadescription) |  | string | Optional. The description of the property. |
+|  [example](./vertexai-preview.schema.md#schemaexample) |  | unknown | Optional. The example of the property. |
+|  [format](./vertexai-preview.schema.md#schemaformat) |  | string | Optional. The format of the property. Supported formats: for NUMBER type: "float", "double" for INTEGER type: "int32", "int64" for STRING type: "email", "byte", etc |
```

Comment:
These need to be separated rather than running on to each other (or be broken out into a bullet list if that's possible).

for NUMBER type: "float", "double"
for INTEGER type: "int32", "int64"
etc.

Reply:
Inserted html list into the comment.